python/ray/serve/tests/unit/test_http_util.py

```diff
@@ -125,9 +125,15 @@ async def test_raises_exception(
         with pytest.raises(RuntimeError, match="oopsies"):
             await asgi_receive_proxy()
 
-    async def test_does_not_raise_key_error(
+    async def test_return_disconnect_on_key_error(
```

Comment:
Nitpick (non-blocker): Can also test the websocket codepath to ensure there won't be future regressions.

Reply:
just added!